src/middlewares/socialCallback.js

```diff
@@ -0,0 +1,27 @@
+import bcrypt from 'bcryptjs';
+import db from '../models';
```

Comment:
let us store as models or we destructure the exact model e.g `import {Users} from '../models';
`